src/sentry/issues/ongoing.py

```diff
@@ -1,48 +1,49 @@
-from typing import Any, Mapping, Optional
+from typing import Any, List, Mapping, Optional
 
 from django.db.models.signals import post_save
 
 from sentry.models import (
-    Activity,
     Group,
     GroupInboxReason,
     GroupStatus,
-    add_group_to_inbox,
-    record_group_history_from_activity_type,
-    remove_group_from_inbox,
+    bulk_add_groups_to_inbox,
+    bulk_remove_groups_from_inbox,
 )
 from sentry.types.activity import ActivityType
 from sentry.types.group import GroupSubStatus
 
 
-def transition_group_to_ongoing(
+def bulk_transition_group_to_ongoing(
     from_status: GroupStatus,
     from_substatus: GroupSubStatus,
-    group: Group,
+    groups: List[Group],
     activity_data: Optional[Mapping[str, Any]] = None,
 ) -> None:
     # make sure we don't update the Group when its already updated by conditionally updating the Group
-    updated = Group.objects.filter(
-        id=group.id, status=from_status, substatus=from_substatus
-    ).update(status=GroupStatus.UNRESOLVED, substatus=GroupSubStatus.ONGOING)
-    if updated:
+    groups_to_transistion = Group.objects.filter(
+        id__in=[group.id for group in groups], status=from_status, substatus=from_substatus
+    )
+
+    Group.objects.update_group_status(
+        groups=groups_to_transistion,
+        status=GroupStatus.UNRESOLVED,
+        substatus=GroupSubStatus.ONGOING,
+        activity_type=ActivityType.AUTO_SET_ONGOING,
+        activity_data=activity_data,
+        send_activity_notification=False,
+    )
+
+    for group in groups_to_transistion:
         group.status = GroupStatus.UNRESOLVED
         group.substatus = GroupSubStatus.ONGOING
+
+    bulk_remove_groups_from_inbox(groups)
+    bulk_add_groups_to_inbox(groups, GroupInboxReason.ONGOING)
```

Comment:
we don't need to put ongoing things in the inbox anymore. since they transition after 7 days